build/sidecar.Containerfile

```diff
@@ -0,0 +1,25 @@
+# Build the manager binary
+FROM golang:1.17 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY sidecar/ sidecar/
+COPY api/ api/
```

Comment:
was there not a shared `internal/` directory too?

Reply:
> was there not a shared `internal/` directory too?


I'll rebase on top #20 when it is merged and add this directory.